mobile/analytics/partners.js

```diff
@@ -2,16 +2,17 @@
   'use strict'
 
   $('.partner-profile-contact-email a').click(function (e) {
-    analytics.track('Clicked Contact Gallery Via Email', {
+    analytics.track('Click',{
       partner_id: $(e.currentTarget).data('partner-id'),
-      partner_slug: $(e.currentTarget).data('partner-slug')
-    })
+      label: "Contact gallery by email"
+    });
```

Comment:
This sends the same event properties as [the desktop event](https://github.com/artsy/force/blob/0dad245/desktop/analytics/partner.js#L27-L30). there may have to be a property that needs to be added to indicate that this is an event from mobile.